packages/block-editor/src/utils/index.js

```diff
@@ -1,3 +1,2 @@
 export { default as transformStyles } from './transform-styles';
 export * from './block-variation-transforms';
-export { default as getPxFromCssUnit } from './parse-css-unit-to-px';
```

Comment:
This is public API. I'm not sure we can just remove this..

Reply:
It's from native, which doesn't have any public APIs?

Reply:
What do you mean from native? See docs: https://developer.wordpress.org/block-editor/reference-guides/packages/packages-block-editor/#getpxfromcssunit

Reply:
This is something that native has added and has no use outside of it. I can add a stub function.

Reply:
It doesn't matter if in core only native uses it. Third party consumers could be using it. We should deprecate instead of removing.